### PR TITLE
Set crown as supported.

### PR DESCRIPTION
**Pull request by sirak92**
Set bitcoin:CRW as supported for trezor1 and trezor2 devices.

### CODE REVIEW

**Comment by matejcik** [comment]
did Crown change their `address_type`? if so, the PR should include that change.

**Comment by sirak92** [comment]
@matejcik did I do something incorrectly as CI build failed or it's an environment issue?

**Comment by matejcik** [comment]
you'll need to run `make gen` in root to regenerate dependent files

also please change the version from "soon" to 2.1.7 for TT and 1.8.4 for T1

**Comment by sirak92** [comment]
After `make gen` git diff shows differences in `tokens.py` and `coins.json`(the patch can't be attached, as the size > 10MB). Do I need to add them also?

**Comment by matejcik** [comment]
coins.json, yes
tokens.py should not be affected. I think you can leave it out, we'll see if the CI passes then.

**Comment by sirak92** [comment]
@matejcik again build failed, should I add the remaining file also?

**Comment by matejcik** [comment]
no, the problem is still in coins.json.
i think you have the tokens submodule missing. run `git submodule update --init --recursive` in repository root and then regenerate with `make gen`, it should fix the situation

**Comment by sirak92** [comment]
indeed the problem was in submodule part

**Comment by walkjivefly** [comment]
@sirak92 this doesn't yet include the updated home and explorer URLs. See https://github.com/Crowndev/trezor-firmware/pull/1

Can you pickup those changes in this PR?

**Comment by sirak92** [comment]
@walkjivefly done.

File: common/defs/bitcoin/crown.json [hunk 1]
```diff
@@ -33,9 +33,9 @@
   "min_address_length": 36,
   "max_address_length": 40,
   "bitcore": [
-    "https://insight-01.crown.tech",
-    "https://insight-02.crown.tech",
-    "https://insight-03.crown.tech"
+    "https://insight-01.crownplatform.com",
```
Comment (prusnak) [inline common/defs/bitcoin/crown.json]: These new bitcore addresses redirect back to crown.tech. Fix that or change these URLs back to crown.tech

**Comment by sirak92** [comment]
hi @prusnak, the changes got accepted more than 3 months ago, but the new firmware for Trezore 1 devices still not released. So I would like to know the date(approximately) when a new release is going to be released?

**Pull request merged**